server/src/main/java/io/crate/types/ParameterTypeSignature.java

```diff
@@ -58,6 +58,7 @@ public TypeSignatureType type() {
 
     @Override
     public String toString() {
-        return parameterName + " " + super.toString();
+        // Quote name as it may be a sub-column ident which is allowed to contain white spaces
+        return "\"" + parameterName + "\" " + super.toString();
```

Comment:
Should we add an entry to the docs, that object column names cannot contain quote symbol `"`? 

Currently it looks like 

>Column names are restricted in terms of patterns:
Columns are not allowed to contain a dot (.), since this conflicts with internal path definitions.
Columns that conflict with the naming scheme of virtual system columns are restricted.
Character sequences that conform to the subscript notation (e.g. col['id']) are not allowed.

Reply:
Good catch, thx! The signature parsing logic should take possible double quotes inside the name into account. I'll come up with a fix for that.